tests/integration/pilot/common/routing.go

```diff
@@ -1747,6 +1770,23 @@ spec:
 `, name, app)
 }
 
+func idletimeoutDestinationRule(name, app string) string {
+	return fmt.Sprintf(`apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: %s
+spec:
+  host: %s
+  trafficPolicy:
+    tls:
+      mode: DISABLE
+    connectionPool:
+      http:
+        idleTimeout: 100s
```

Comment:
are we actually testing the idle timeout? It seems we just make sure this doesn't break things entirely - which I guess is fine but maybe we can comment that to avoid making the test seem like its doing more than it is?

I don't have any good ideas to fully test the field

Reply:
No. I was not thinking of validating idle timeout behaviour.  It is meant to validate config is generated correctly. Added comment.